mysticeti-core/src/block_handler.rs

```diff
@@ -37,6 +37,8 @@ pub trait BlockHandler: Send + Sync {
 
     fn recover_state(&mut self, _state: &Bytes);
 
+    fn transaction_time(&self) -> Arc<Mutex<HashMap<TransactionLocator, TimeInstant>>>;
```

Comment:
Why do we have to expose this to the interface? This is really something specific to benchmark block handler only